ee/models/property_definition.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+from posthog.models.property_definition import PropertyDefinition
+
+
+class EnterprisePropertyDefinition(PropertyDefinition):
+    description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    tags: ArrayField = ArrayField(models.CharField(max_length=32), null=True, default=list)
```

Comment:
Probably worth keeping the updated_at & updated_by if it's not a big deal, might end up being helpful.